addons/l10n_it_edi_doi/models/res_company.py

```diff
@@ -0,0 +1,36 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _name = 'res.company'
+    _inherit = 'res.company'
+
+    l10n_it_edi_doi_tax = fields.Many2one(
+        comodel_name='account.tax',
+        compute='_compute_l10n_it_edi_doi_tax',
+        string="Declaration of Intent Tax",
+    )
+
+    l10n_it_edi_doi_fiscal_position = fields.Many2one(
+        comodel_name='account.fiscal.position',
+        compute='_compute_l10n_it_edi_doi_fiscal_position',
+        string="Declaration of Intent Fiscal Position",
+    )
+
+    def _compute_l10n_it_edi_doi_fiscal_position(self):
+        it_companies = self.filtered(lambda company: company.chart_template == 'it')
```

Comment:
```suggestion
        it_companies = self.filtered(lambda company: company.account_fiscal_country_id.code == 'IT')
```
Same below.

Reply:
Function does not exist anymore; see previous comment